### PR TITLE
Task yc/j386 make background white again

### PR DESCRIPTION
**Pull request by EugeniusGR**
image
image

### CODE REVIEW

**Review by Denidzio — CHANGES_REQUESTED** [review]
It will be better if we make `background-color` once for each screen through react-navigation.

**Comment by EugeniusGR** [comment]
> It will be better if we make `background-color` once for each screen through react-navigation.

thats possible when you are using StackNavigation (as we do, and if you look at the code, you will see those lines `cardStyle: { backgroundColor: '#fff' },` , but thats impossible to do with our case, when the tab navigation is inside stack navigation. Tab navigation has this property but we must create container, what is impossible to do with our case

**Comment by Denidzio** [comment]
> > It will be better if we make `background-color` once for each screen through react-navigation.
> 
> thats possible when you are using StackNavigation (as we do, and if you look at the code, you will see those lines `cardStyle: { backgroundColor: '#fff' },` , but thats impossible to do with our case, when the tab navigation is inside stack navigation. Tab navigation has this property but we must create container, what is impossible to do with our case

:(

**Comment by EugeniusGR** [comment]
new button design
image

File: src/screens/Home/styles.ts [hunk 1]
```diff
@@ -15,7 +15,8 @@ type Styles = {
 
 const styles = StyleSheet.create<Styles>({
   container: {
-    padding: 25
+    padding: 25,
+    backgroundColor: '#fff'
```
Comment (eugeneprib) [inline src/screens/Home/styles.ts]: You could just move this as a global styles variable

**Review by eugeneprib — CHANGES_REQUESTED** [review]
Make tabs background also white

**Pull request merged**